### DISCUSSION TITLE
Why does Github allow and facilitate the blatant copyright infringements of books?

### DISCUSSION THREAD

**Discussion by brianstorms** [community/community#109591] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I was dismayed to discover yesterday that for four years, my nonfiction computer history book, which I worked on for 32 years before it was published by Pantheon Books in 2017, has been sitting on Github as an EPUB that anyone can download and read for free. As the copyright owner of this published work, this sucks. Why does Github allow this to take place? My book isn't the only thing on here; there are tons of other EPUB book files online as well.

**Comment by ettaboyle** [comment]
Thank you kindly for reaching out. If you are experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam We hope that helps!

**Reply by brianstorms** [reply]
This isn't resolved at all.

**Comment by Soren-365** [comment]
What a joke.  Github breaking copyright infringement for years and years on end, and are not hold accountable.
Well i guess fodder is needed for ChatGpt and OpenAI among others.   Billions invested in Nvidia, but copyright holders are f*ed.

**Reply by jrledurt** [reply]
I think copyright law has a precedent set where authors have to request takedown notices.  It's like this for music. I see music illegally uploaded to Spotify all the time.  I mean the guy/publisher should be knowledgable about this.  It's kinda messed up but you want GitHub to personally analyze every file that gets uploaded? would be impossible.  And if GitHub could be sued for every infringement it wouldn't exist. neither would any cloud file hosting site in the US.  I know they do like file matching bit for bit type stuff but that is easily workaround able.
